### PR TITLE
Pong

### PR DESCRIPTION
**Pull request by Sparky200**
The jar file will open a pong game when run manually as an executable.

### CODE REVIEW

File: src/main/java/pong/Pong.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        var mid = new Dimension(rootPane.getSize().width / 2, rootPane.getSize().height / 2);
+        ball.setLocation(mid.width - 10, mid.height - 10);
+        playerPaddle.setLocation(20, mid.height - 50);
+        aiPaddle.setLocation(mid.width * 2 - 40, mid.height - 50);
+
+        final int[] addPlayerVel = {0};
+
+        rootPane.addKeyListener(new KeyAdapter() {
+            private final boolean[] fuckYouJava = new boolean[2];
+
```
Comment (Fusezion) [inline src/main/java/pong/Pong.java:75]: Good naming, nothing wrong here very much agree

**Review by Fusezion — APPROVED** [review]
Would play again

**Review by ShaneBeee — APPROVED** [review]
Looks good to me

**Comment by ShaneBeee** [comment]
Thank you for your contribution 👍🏻

**Pull request merged**